app/views/katello/api/v2/activation_keys/product_content.json.rabl

```diff
@@ -8,11 +8,22 @@ child @collection[:results] => :results do
   end
 
   child :content => :content do
-    attributes :id, :name, :label
+    attributes :id, :name, :label, :contentUrl => :content_url
```

Comment:
This doesn't work in a list of attributes, I guess you can't rename in a list of attributes?

You have to do this for it to work:

```
attributes :id, :name, :label
attribute :contentUrl => :content_url
```